src/backend/utils/storage.js

```diff
@@ -45,6 +45,16 @@ module.exports = {
       .exec();
   },
 
+  removeFeedById: async id => {
+    // Remove the feed key itself (t:feed:{feed-id})
+    await redis.del(`${feedsKey}:${id}`);
```

Comment:
> You can move this `redis.del ...` call down inside the `.multi() ... .exec()` chain. Those commands bookend a series of Redis steps you want to make atomic.
> 
> Also, don't manually construct the feed key, use `createFeedKey(id)`

Good calls, done in https://github.com/Seneca-CDOT/telescope/pull/727/commits/77ecd048a8792fd4af8697886eaba29bba8d1627

>I'm realizing as I read this that we're going to run into an issue very soon as we start adding users. We won't be able to use the whole set of feeds to compare the wiki feeds against, since it will include feeds that were added by other users.

I see...

> I think we're going to have to add feed ownership before we can finish this fix.

Is this currently represented in any open issue? If not, I would be happy to open an issue to implement 'feed ownership' once I get a better idea of exactly what changes need to be made to do so, e.g. in terms of Redis sets, etc.

Reply:
#706 is part of it, and I think @c3ho has filed others (but I don't have them off the top of my head).

Reply:
Okay, perfect. I'll hold out hope that this can be tackled within the current Release, and I'll be in the market for any pieces of those issues that I can contribute to in lieu of this one.